tests/models/test_text_message.py

```diff
@@ -56,6 +58,36 @@ def test_null_emojis(self):
             TextMessage(**arg).as_json_dict()
         )
 
+    def test_mention(self):
+        arg = {
+            "type": "text",
+            "text": "@example Hello, world! (love)",
+            "mention": {
+                "mentionees": [
+                    Mentionee(
+                        index=0,
+                        length=8,
+                        user_id="U850014438e..."
+                    )
+                ]
+            },
+        }
+        self.assertEqual(
+            self.serialize_as_dict(arg, type=self.TEXT),
+            TextMessage(**arg).as_json_dict(),
+        )
+        self.assertEqual(
+            TextMessage(**arg).mention,
+            Mention(mentionees=arg["mention"]["mentionees"])
+        )
```

Comment:
I tried to use `Mention` object for` arg`.
But `serialize_as_dict` didn't work.
Please let me know if I make a mistake.

<img width="407" alt="スクリーンショット 2021-01-30 1 53 14" src="https://user-images.githubusercontent.com/49806926/106303957-1add5600-629e-11eb-86f9-95f8ca086472.png">
<img width="655" alt="スクリーンショット 2021-01-30 1 53 28" src="https://user-images.githubusercontent.com/49806926/106303961-1c0e8300-629e-11eb-8dbb-30c0827b61dd.png">

Reply:
With this change, the `test` will succeed, but it will fail to receive the event.

<img width="921" alt="スクリーンショット 2021-01-30 2 23 13" src="https://user-images.githubusercontent.com/49806926/106306994-20d53600-62a2-11eb-909a-ecfe8841ab65.png">

<img width="393" alt="スクリーンショット 2021-01-30 2 20 24" src="https://user-images.githubusercontent.com/49806926/106306702-c340e980-62a1-11eb-9541-0ad1d17c37e0.png">
<img width="1013" alt="スクリーンショット 2021-01-30 2 24 06" src="https://user-images.githubusercontent.com/49806926/106307093-42ceb880-62a2-11eb-8b99-6cfa4a999bb9.png">

<img width="686" alt="スクリーンショット 2021-01-30 2 20 34" src="https://user-images.githubusercontent.com/49806926/106306709-c50aad00-62a1-11eb-88c0-14bb97b94123.png">

Reply:
This part is my communication error.

1. Because we have `Mention` object, so we have `get_or_new_from_json_dict` to get Mention if developers use the object.

```
        if mention:
            mention_object = self.get_or_new_from_json_dict(
                    mention, Mention
            )
            mentionees = []
            for mentionee in mention_object["mentionees"]:
                mentionee_object = self.get_or_new_from_json_dict(
                    mentionee, Mentionee
                )
                if mentionee_object:
                    mentionees.append(mentionee_object)
            self.mention = Mention(mentionees)
        else:
            self.mention = mention
```


2. and your test case just like the following code:

```
    def test_mention(self):
        arg = {
            "type": "text",
            "text": "@example Hello, world! (love)",
            "mention": {
                "mentionees": [{
                    "index": 0,
                    "length": 8,
                    "user_id": "U850014438e..."
                }]
            },
        }
        self.assertEqual(
            self.serialize_as_dict(arg, type=self.TEXT),
            TextMessage(**arg).as_json_dict(),
        )
```

As I see, that would be better in this case.

Reply:
Oh, I see.
thank you. fixed.